foerder.html

```diff
@@ -439,8 +458,9 @@
 				document.getElementsByName('country')[0].value = document.getElementsByName('sc-country')[0].value;
 				generateQR()
 
-				document.getElementById('sanityChecker').style.display = "none";
+				document.getElementById('sanityChecker').style.height = "0";
```

Comment:
Together with the changes to the #sanityChecker css, this makes the sanityChecker div scroll out when clicking the button. It triggers the animation, while display none is an instant change.